src/controllers/resume.submission.controller.ts

```diff
@@ -31,11 +33,28 @@ export const get = async (req: Request, res: Response, next: NextFunction) => {
     if (isActiveFeature(config.FEATURE_FLAG_ENABLE_SAVE_AND_RESUME_17102022)) {
       const appData: ApplicationData = await getOverseasEntity(req, transactionId, overseaEntityId);
 
-      if (!appData || !Object.keys(appData).length) {
+      if (!Object.keys(appData || {}).length) {
         throw createAndLogErrorRequest(req, `Error on resuming OE - ${infoMsg}`);
       }
 
-      setWebApplicationData(req.session as Session, appData, transactionId, overseaEntityId);
+      const session = req.session as Session;
+      setWebApplicationData(session, appData, transactionId, overseaEntityId);
+
+      const resource = await getTransaction(req, transactionId);
+
+      // https://companieshouse.atlassian.net/wiki/spaces/TTL/pages/4063854602/Pay+Now+link+Resume
```

Comment:
I don't think we need the link to Confluence in the code. As long as it's referenced on the related JIRA ticket we are fine.

Reply:
I will move it on the description of the following PR, JIRA sometime can be very difficult/messy